tensorboard/backend/application_test.py

```diff
@@ -339,5 +339,23 @@ def testExceptions(self):
       application.TensorBoardWSGIApp(logdir, plugins, multiplexer, 0)
 
 
+class DbTest(tf.test.TestCase):
```

Comment:
There's no easy way to test that particular piece of glue. It's better left to future integration tests.